Steps/UploadCodeCoverage.yml

```diff
@@ -0,0 +1,31 @@
+## @file
+# Azure Pipelines step template upload code coverage to codecov.io
+#
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+parameters:
+- name: codecov_token
+  displayName: Codecov.io Token
+  type: string
+  default: ''
+- name: report_dir
+  displayName: Code Coverage Report
+  type: string
+  default: ''
+
+steps:
+- powershell: |
```

Comment:
While we're trying to reduce dependencies on Powershell overall, I'm fine with this since its usage is conditionalized to Windows. Another option would be to write a single step in Python where it detects the host OS.

Feel free to close though if you don't want to do that.

Reply:
I did move to python.